package.json

```diff
@@ -45,9 +46,9 @@
     "check-eslint": "eslint --quiet .",
     "checkout-dev": "lerna exec 'git checkout dev' --parallel --no-bail",
     "clean-node-modules": "npx rimraf node_modules && npx rimraf package-lock.json && npx lerna exec npx rimraf node_modules && npx lerna exec npx rimraf package-lock.json",
-    "clone-project": "cross-env ts-node --swc scripts/clone-project.ts",
-    "create-root-package-json": "cross-env ts-node --swc scripts/create-root-package-json",
-    "create-project": "cross-env ts-node --swc scripts/create-project",
+    "clone-project": "cross-env vite-node scripts/clone-project.ts",
+    "create-root-package-json": "cross-env vite-node scripts/create-root-package-json",
+    "create-project": "cross-env vite-node scripts/create-pr  oject",
```

Comment:
Is this `create-pr  oject` part intentional or typo?

Reply:
Does not look intentional :)

Reply:
Nope. Don't know how that happened.